mev/api/converters/data_resource.py

```diff
@@ -352,46 +353,46 @@ def _create_resource(self, executed_op, workspace, path, name):
-        '''
-        Handles the conversion of a UUID referencing an instance that
-        is a child of api.models.AbstractResource 
-
-        Returns the full path to the file, e.g.
-        s3://<mev storage bucket>/<object path>
-        '''
-        r = get_resource_by_pk(resource_uuid)
-        return default_storage.get_absolute_path(r.datafile.name)
-
-    def _create_resource(self, executed_op, workspace, path, name):
-        '''
-        Returns an instance of api.models.Resource based on the 
-        passed parameters.
-
-        Note that `path` is the path in the Cromwell bucket and 
-        we have to move the file into WebMeV storage.
-        '''
-        logger.info('From executed operation outputs based on a Cromwell-based'
-                    f' job, create a resource with name {name}')
-        try:
-            r = default_storage.create_resource_from_interbucket_copy(
-                executed_op.owner,
-                path
-            )
-            if workspace is not None:
-                r.workspaces.add(workspace)
-            return r
-        except Exception as ex:
-            logger.info('Caught exception when copying a Cromwell output'
-                        ' to our storage. Removing the dummy Resource'
-                        ' and re-raising.')
-            raise ex
+# class CromwellResourceMixin(object):
+#     '''
+#     Contains behavior for DataResource conversion related to
+#     the Cromwell runner.
+#     '''
+
+#     def _convert_resource_input(self, resource_uuid, staging_dir):
+#         '''
+#         Handles the conversion of a UUID referencing an instance that
+#         is a child of api.models.AbstractResource 
+
+#         Returns the full path to the file, e.g.
+#         s3://<mev storage bucket>/<object path>
+#         '''
+#         r = get_resource_by_pk(resource_uuid)
+#         return default_storage.get_absolute_path(r.datafile.name)
+
+#     def _create_resource(self, executed_op, workspace, path, name):
+#         '''
+#         Returns an instance of api.models.Resource based on the 
+#         passed parameters.
+
+#         Note that `path` is the path in the Cromwell bucket and 
+#         we have to move the file into WebMeV storage.
+#         '''
+#         logger.info('From executed operation outputs based on a Cromwell-based'
+#                     f' job, create a resource with name {name}')
+#         try:
+#             r = default_storage.create_resource_from_interbucket_copy(
+#                 executed_op.owner,
+#                 path
+#             )
+#             if workspace is not None:
+#                 r.workspaces.add(workspace)
+#             return r
+#         except Exception as ex:
+#             logger.info('Caught exception when copying a Cromwell output'
+#                         ' to our storage. Removing the dummy Resource'
+#                         ' and re-raising.')
+#             raise ex
```

Comment:
It's a best practice to delete unused code instead of commenting it out.

Reply:
yeah, i opted for this so i can go back later and tweak it for nextflow since the logic will largely be similar. Thought it would save me some time rather than going back and finding all the code i previously removed. Once we have nextflow implemented and the everything is 'put back together' these will be fixed and uncommented so that it won't be there when finally squashed and merged.